parlai/mturk/core/dev/react_server/dev/components/socket_handler.jsx

```diff
@@ -438,56 +379,51 @@ class SocketHandler extends React.Component {
       }
       this.setState({ message_request_time: new Date().getTime() });
       log('Waiting for worker input', 4);
-    } else if (command === COMMAND_SHOW_DONE_BUTTON) {
-      // Update the UI to show the done button
-      this.props.onTaskDone();
-    } else if (command === COMMAND_INACTIVE_DONE) {
-      // Update the UI to show done with additional inactive text
-      this.closeSocket();
-      // Call correct UI renderers
-      this.props.onInactiveDone(msg['inactive_text']);
     } else if (command === COMMAND_SUBMIT_HIT) {
       // Force the hit to submit as done
       this.props.onForceDone();
-    } else if (command === COMMAND_EXPIRE_HIT) {
-      // Expire the hit unless it has already been marked as done
-      if (!this.props.task_done) {
-        this.props.onExpire(msg['inactive_text']);
-        this.closeSocket();
-      }
-    } else if (command === COMMAND_INACTIVE_HIT) {
-      // Disable the hit, show the correct message
-      this.props.onExpire(msg['inactive_text']);
-      this.closeSocket();
-    } else if (command === COMMAND_RESTORE_STATE) {
-      // Restore the messages from inside the data, call the command if needed
-      let messages = msg['messages'];
-      for (var i = 0; i < messages.length; i++) {
-        this.handleNewMessage(messages[i]['message_id'], messages[i]);
-      }
+    } else if (command === COMMAND_USER_DEFINED) {
```

Comment:
just checking that this empty block does not cause any errors in js

Reply:
Nope, JS doesn't care